microcosm_flask/swagger/parameters/enum.py

```diff
@@ -59,7 +59,7 @@ def parse_type(self, field: Field) -> str:
             raise Exception(f"Cannot infer enum type for field: {field.name}")
 
     def _parse_enum_values(self, field: Field) -> Sequence:
-        enum = getattr(field, "enum", None)
+        enum = getattr(field, "enum")
```

Comment:
Unrelated change to fix typehinting. Not that in the `None` case line 63 would have errored anyway, so this change doesn't introduce a new risk – this code is only reached when `self.strict_enums` is true (see line 69), in which case the `"enum"` attribute should be present.

Reply:
`field.enum` then?